Lang/en/commands.json

```diff
@@ -460,6 +460,20 @@
     "success": "Pet nickname changed to **{{name}}**!",
     "successNoName": "Your pet no longer has a nickname!"
   },
+  "classesInfo": {
+    "title": {
+      "list": ":bookmark: Classes list",
```

Comment:
l'emote est à décaler dans DraftbotIcons

Reply:
Faudra donc aussi décaler les emotes des autres commandes là-bas, dans ce cas. C'est pas le seul endroit où il y a l'emote dans le JSON des commandes, *même en français*.

Reply:
its time to create an issue then !

Reply:
En vrai pas spécialement besoin de déplacer tous ceux dans les fichiers de trad. Il n'y a besoin que quand c'est utilisé à plusieurs endroits

Reply:
le soucis c'est que : 
1- les traducteurs peuvent traduire par mégarde les emotes, c'est con mais ca peut arriver
2- les emotes dans les textes ne seront pas réutilisables autre part que dans discord, donc si futur app, faudra retraduire nous mêmes les emotes discord

Reply:
C'est vrai

Reply:
est-ce que l'issue a été créée au final ?